src/lib/cht-session.ts

```diff
@@ -31,13 +31,15 @@ export default class ChtSession {
   public readonly axiosInstance: AxiosInstance;
   public readonly sessionToken: string;
   public readonly chtCoreVersion: string;
+  public readonly isAdmin: boolean;
 
   private constructor(creationDetails: SessionCreationDetails) {
     this.authInfo = creationDetails.authInfo;
     this.username = creationDetails.username;
     this.facilityIds = creationDetails.facilityIds;
     this.sessionToken = creationDetails.sessionToken;
     this.chtCoreVersion = creationDetails.chtCoreVersion;
+    this.isAdmin = this.facilityIds.includes(ADMIN_FACILITY_ID);
```

Comment:
dont love this change.  you're trading off space vs time complexity (caching) for something that doesnt really need to be cached... it appears to be called in very few locations.  can you explain your rationale?